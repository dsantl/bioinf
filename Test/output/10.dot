17 14 6.244522
17 16 3.450465
17 15 6.111864
14 1 3.230184
14 10 6.277632
10 2 -0.491675
10 6 3.657146
15 0 12.293224
15 13 7.733812
13 8 10.630121
13 12 2.258506
12 4 -5.041282
12 7 7.818956
16 5 10.104486
16 11 5.763650
11 3 0.222993
11 9 6.787598
