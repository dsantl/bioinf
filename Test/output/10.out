17 11 5.763650
11 3 0.222993
11 9 6.787598
17 5 10.104486
17 16 3.450465
16 15 6.111864
15 0 12.293224
15 13 7.733812
13 12 2.258506
12 4 -5.041282
12 7 7.818956
13 8 10.630121
16 14 6.244522
14 1 3.230184
14 10 6.277632
10 2 -0.491675
10 6 3.657146
